### PR TITLE
feat: add contributors page

### PR DESCRIPTION
**Pull request by wwisser**

### CODE REVIEW

**Comment by wwisser** [comment]
its basic and backlinks are missing on the other pages

**Comment by wwisser** [comment]
IMO we should also make a component for the navigation section for DRY rule

File: src/app/contributors/page.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        </li>
+                        <li>
+                            <Link href="/hacks" className="hover:text-gray-400">
+                                Hacks
+                            </Link>
+                        </li>
+                        <li>
+                            <Link href="/blog" className="hover:text-gray-400">
+                                Blog
+                            </Link>
```
Comment (wwisser) [inline src/app/contributors/page.tsx:35]: Let's make a component out of this so we have it reusable.

File: src/app/contributors/page.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import Link from "next/link";
+
+async function getContributors() {
+    const res = await fetch("https://api.github.com/repos/TPOT-Hacks/website/contributors", {
```
Comment (wwisser) [inline src/app/contributors/page.tsx:4]: If we want to have this across the whole organization, we have to change this call.

File: src/app/contributors/page.tsx (hunk 1 above)
Comment (wwisser) [reply src/app/contributors/page.tsx:35]: And add the new page of course!

File: src/app/contributors/page.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                            </svg>
+                            Back to Hacks
+                        </Link>
+                    </div>
+
+                    {/* GitHub Contributors Section */}
+                    <section className="mt-16">
+                        <h2 className="text-2xl font-bold mb-4">GitHub Contributors</h2>
+                        <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-4">
+                            {contributors.map((contributor: any) => (
```
Comment (wwisser) [inline src/app/contributors/page.tsx:72]: We should interface this type correctly, if we want to have a clean implementation.

**Comment by KamdynS** [comment]
This is the right start. Let's get backlinks setup so it's fully functional. I'll merge after that. Be sure you are trying to merge with the `staging` branch though. Don't make PRs directly to `main`.

**Pull request merged**